plugin/web/components/layout/WdContent.tsx

```diff
@@ -0,0 +1,45 @@
+import { Content } from "antd/lib/layout/layout";
+import React,{ memo, ReactNode } from "react";
+
+import "../../../node_modules/antd/dist/antd.less";
+import InfoTab from "../Tabs/InfoTab";
+import MapSettingTab from "../Tabs/MapSettingTab";
+import ShareTab from "../Tabs/ShareTab";
+
+type Props = {
+  className?: string;
+  children?: ReactNode;
+  current: string;
+};
+
+//  {
+//     key: "mapData",
+//   },
+//   {
+//     key: "mapSetting",
+//   },
+//   {
+//     key: "shareNprint",
+//   },
+//   {
+//     key: "about",
+//   },
+//   {
+//     key: "template",
+//   },
+
+const WdContent: React.FC<Props> = ({ className, current, children }) => {
+  return (
+    <Content className={className}>
+      {
+        {
+          mapSetting: <MapSettingTab />,
+          shareNprint: <ShareTab />,
+          about: <InfoTab />,
+        }[current]
```

Comment:
Can you explain this part of code? I have never seen something like this before and checking the antd documentation I can see it either

Reply:
this something like inline enum , you can use it instead of if , else conditions or switch